### DISCUSSION TITLE
Remove -Delete Old Devices not working or avaiable

### DISCUSSION THREAD

**Discussion by thorrrr** [alandtse/alexa_media_player#1288] [Q&A] [answered]
HI 
I love the app but i renewed 8 echos and removed my old ones from Amazon.
But even with the latest update i still have the old ones in my instance of HA.
I know you can hide them but i am not able to delete them and i cannot use the entity names without renaming the old ones.
Is there any way to fix this please?
![Current Screenshot 87](https://user-images.githubusercontent.com/8170673/118135976-838ab180-b3fb-11eb-913f-b150c8214369.jpg)

**Comment by alandtse** [comment, marked as answer]
This is resolved in 3.10.1 and above.

**Reply by thorrrr** [reply]
Hi Alan
I have updated to latest version but still seeing old entries  and cannot remove them
![AlexaOldEntries](https://user-images.githubusercontent.com/8170673/118788990-393a8200-b88c-11eb-83c9-bbeb54ab835b.jpg)

**Reply by alandtse** [reply]
Report a bug using the issues tab.

**Reply by GizmoGoody** [reply]
@thorrrr Did you ever resolve this? I have the same issue.

**Reply by thorrrr** [reply]
Please close
